packaging/README.md

```diff
@@ -1,35 +1,85 @@
 This is mostly notes to myself.
 
+# Refs
+
+Some links and refs useful to keep handy
+
+* [Sugar Jar dist-git](https://src.fedoraproject.org/rpms/rubygem-sugarjar)
+* [Package Maintenance Guide](https://docs.fedoraproject.org/en-US/package-maintainers/Package_Maintenance_Guide/)
+* [Machines you can use](https://fedoraproject.org/wiki/Test_Machine_Resources_For_Package_Maintainers)
+
+# Prep
+
 Start the vagrant machine, ssh to it (`vagrant up; vagrant ssh`)
 
-In the mounted repo, create the source file for the tests, see the comment in
-the spec file
+If not already checked out, check out the dist-git:
+
+```shell
+fedpkg co rubygem-sugarjar
+```
+
+If already checked out, do `fedpkg pull` to get the latest.
+
+# Do work
 
-Then in another dir (ala ~/builddir or whatever), copy the specfile and the
-test source file in, as well as wget the gem file from the URL in the spec
-file.
+Make whatever changes you want on rawhide.
 
-Use fedpkg to get a srpm (you may have to install various deps):
+If you're doing a version bump you'll need to grab both new sources and replace
+the old ones. First follow the directions in the spec file to build the tarball
+for the test files. Then wget the gem from the URL in the spec file. Then:
 
 ```shell
-fedpkg --release rawhide local
+fedpkg new-sources rubygem-sugarjar-<version>-specs.tar.gz sugarjar-<version>.gem
 ```
 
-Use koji to build:
+# Testing
+
+You can do a local build (`fedpkg local`) or a mock build (`fedpkg mockbuild`).
+
+You can, alternatively, submit a koji build:
 
 ```shell
+# build a SRPM
+fedpkg srpm
+# make sure your krb-auth'd
 krb
+# Submit the koji build
 koji build --scratch rawhide <srpm>
 ```
 
-Download and install the RPM
+# Committing and pushing
 
-Request review [here](https://bugzilla.redhat.com/bugzilla/enter_bug.cgi?product=Fedora&format=fedora-review)
+First, commit your change:
+
+```shell
+fedpkg commit
+```
 
-Or use fedora-create-review once you have access to the system
+You can push directly to master if you want (`fedpkg push`), or alternatively,
+make a PR by adding your remote:
+
+```shell
+git remote add fork ssh://jaymzh@pkgs.fedoraproject.org/forks/jaymzh/rpms/rubygem-sugarjar.git
+```
+
+And push to that instead (`git push fork`), and click the link to make a PR.
+
+Once it's pushed/merged, you can create a build:
+
+```shell
+fedpkg build
+```
+
+For Rawhide, if the build succeeds, you're done.
+
+To build for other distros, switch branches with:
+
+```shell
+fedpkg switch-branch <f35,f36,etc.>
+```
 
-after requesting review you can check it yourself with:
+and repeat. For non-rawhide branches, after the `build`, submit the update:
```

Comment:
FYI for non-rawhide branches you can do 'git merge rawhide' and just keep them all in sync, unless for some reason they need to deviate